lib/stsci/tools/asnutil.py

```diff
@@ -14,7 +14,9 @@
 from astropy.io import fits
 import numpy as N
 import os.path, time
+from distutils.version import LooseVersion
 
+ASTROPY_VER_GE13 = LooseVersion(astropy.__version__) >= LooseVersion('1.3')
```

Comment:
This is tricky. Because it depends on whether your dev version is before or after the PR that added the deprecation warning.